package.json

```diff
@@ -10,7 +10,7 @@
   "private": true,
   "packageManager": "npm@8.5.0",
   "engines": {
-    "node": ">=16.8.0",
+    "node": ">=14",
```

Comment:
@dvargas92495 can you please explain why is this necessary?

Reply:
https://aws.amazon.com/about-aws/whats-new/2022/05/aws-lambda-adds-support-node-js-16/

Now I'm definitely going to revert this back, I can finally move to node 16 locally!

Reply:
Right on time!